R/msaType1Gauge.R

```diff
@@ -150,6 +150,11 @@ msaType1Gauge <- function(jaspResults, dataset, options, ...) {
 
   table <- createJaspTable(title = gettext("t-Test of Observed Bias Against 0"))
 
+  if (nrow(dataset[measurements]) < 2){
+    table$setError(gettextf("t-Test requires more than 1 measurement. %i valid measurement(s) detected in %s.", nrow(dataset[measurements]), measurements))
```

Comment:
Shouldn't this be `t-test` or `T-test`? Or just `The t-test` to avoid the capitalization issue altogether.

Reply:
In the T-Tests module it is "T-Test" in the table titles and "t-test" in the footnotes, so I will change it to that.